tests/integration_tests/tests/agentless_tests/test_component_type.py

```diff
@@ -100,3 +94,130 @@ def test_component_creation_with_not_existing_blueprint_package(self):
                           self.deploy_application,
                           dsl_path,
                           deployment_id=deployment_id)
+
+
+class ComponentScaleCreation(AgentlessTestCase):
+    component_name = 'component'
+
+    def test_given_deployment_name_with_auto_suffix_inc_option(self):
```

Comment:
The test is called `...auto_suffix_inc...` yet the option in the blueprint is called `auto_inc_suffix`.
ofc it can be either but it should be the same

Reply:
I prefer the former :D